### PR TITLE
Documentation refresh after the migration

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just an update towards publishing a new version from the new location

- [x] Update the documentation after moving the repository
- [x] Be explicit about the officialWiremock images and `/holomekc/wiremock-gui`

@gitkent would you mind if I also change the default image to `wiremock/wiremock` while keeping the current one documented?

### CODE REVIEW

**Review by gitkent — APPROVED** [review]
lgtm

On a side note: The CI job has failed due to it is expecting a version bump on the Chart version. What we can possibly do is only run this check when there is any helm chart files changed. Are you happy to make this change or you can remove the "Required" for `CI/lint-test` for now, I will create a separate PR to fix it.

**Comment by oleg-nenashev** [comment]
Hi! Sorry, missed the comment.

File: charts/wiremock/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,23 @@
 apiVersion: v1
 appVersion: "2.32.0.2"
-description: A Helm chart for Wiremock deployment on Kubernetes
+description: A Helm chart for WireMock deployment on Kubernetes
 name: wiremock
-version: 0.1.3
+version: 0.1.4
+maintainers:
+  - name: "Kent Lee"
```
Comment (oleg-nenashev) [inline charts/wiremock/Chart.yaml:7]: I hope that's fine @gitkent . It is visible from public email but it can be anonymized if you prefer

Comment (oleg-nenashev) [reply charts/wiremock/Chart.yaml:7]: Uhm,, does not work anyway:

```
 ✖︎ wiremock => (version: "0.1.4", path: "charts/wiremock") > Error validating maintainer 'Kent Lee': 404 Not Found
```

Comment (oleg-nenashev) [reply charts/wiremock/Chart.yaml:7]:
```suggestion
  - name: "gitkent"
```

File: charts/wiremock/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+    url: "https://github.com/gitkent"
+  - name: "Oleg Nenashev"
```
Comment (oleg-nenashev) [inline charts/wiremock/Chart.yaml:9]:
```suggestion
  - name: "oleg-nenashev"
```

**Comment by gitkent** [comment]
> Just an update towards publishing a new version from the new location
> 
> - [x] Update the documentation after moving the repository
> - [x] Be explicit about the officialWiremock images and `/holomekc/wiremock-gui`
> 
> @gitkent would you mind if I also change the default image to `wiremock/wiremock` while keeping the current one documented?
> 

@oleg-nenashev yup. That's fine.

**Review by gitkent — APPROVED** [review]

**Pull request merged**